### PR TITLE
feat(docs): added date and demo Widget use cases

### PR DESCRIPTION
**Pull request by zdrazil**
Adds two use cases: connecting date inputs and how to use test environment (this is one is really small, not sure if it should count).

I didn't use CodePen, since this way it should be easier to maintain and readers can easily see the whole HTML document even with `head` tag.

#### Changelog notes 

I don't think this needs changelog.

```
* Added/Extended operations....
```

#### Check during review

- [ ] JSON example extended.
  - [ ] New properties are added to the correct place in the JSON.
- [ ] New properties in the table are added to the correct place.
- [ ] Correct formatting:
  - [ ] Spacing is consistent between titles, sections, tables, ...
  - [ ] Correct JSON format - indentation.
- [ ] DateTime properties should always be defined in ISO format.

### CODE REVIEW

File: distributor-widget/use-cases/README.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Distributor Widget use cases
+
+This section describes, how to use the Distributor Widget in some well-known scenarios:
+
+* [Testing in demo/staging/testing environment](./testing-in-staging-environment.md)
+* [Connect to date inputs](./connect-to-date-inputs.md)
```
Comment (koprivajakub) [inline distributor-widget/use-cases/README.md:6]: I would name this differently:

e.g. Use your own Date inputs

File: distributor-widget/use-cases/connect-to-date-inputs.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+        <script>
+            // 3. Initialize Distributor Widget just before the closing </body> tag
+            Mews.Distributor(
+                // 3.1 Set configuration id of your Distributor
+                {
+                    configurationIds: ["Your Distributor configuration id"],
+                },
+                // Add callback which will enable Submit button and open Distributor Widget upon button click.
+                function (dst) {
```
[the remark this answers is not in the document]
Comment (koprivajakub) [reply distributor-widget/use-cases/connect-to-date-inputs.md:42]: what about name it `api` ?

File: distributor-widget/use-cases/connect-to-date-inputs.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+
+If you are not sure where to find the configuration id, see [getting started section for initializing Distributor Widget](../getting-started.md#initialize-distributor-widget) for details.
+
+### 4. Optional dataBaseUrl parameter
+
+{% hint style="warning" %}
+By default Distributor Widget uses production environment and data. If this is what you want, you can ignore this section and not set `dataBaseUrl` in third parameter.
+{% endhint %}
+
+But if you want to use Distributor Widget with demo data, you need to set [dataBaseUrl](../reference.md#string-databaseurl). You can see more details in [testing in demo/staging/testing environment section](./testing-in-staging-environment.md).
```
Comment (koprivajakub) [inline distributor-widget/use-cases/connect-to-date-inputs.md:102]: I would rephrase it. (including the title)

This guide is written for production environment, if you want to test it during development please refer to our guide how to...

File: distributor-widget/use-cases/testing-in-staging-environment.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+Most of the steps are the same as in [Getting started section](../getting-started.md).
+
+The only difference is in the [initialization of the Distributor Widget](../getting-started.md#initialize-distributor-widget).
+
+In comparison to the default example, you can set an [optional `dataBaseUrl`](../reference.md#string-databaseurl) property to the testing/staging API URL:
+
+```javascript
+Mews.Distributor(
+    { configurationIds: ['aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeeeee'] },
+    function (dst) { dst.open(); },
```
Comment (koprivajakub) [inline distributor-widget/use-cases/testing-in-staging-environment.md:16]: again, I would name it `api`

File: distributor-widget/use-cases/testing-in-staging-environment.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+    { dataBaseUrl: 'https://api.mews-demo.com' }
+);
+```
+
+After doing this, Distributor Widget will start using the data from [staging/testing environment](../../distributor-api-v1/environments.md), instead of from the production.
```
Comment (koprivajakub) [inline distributor-widget/use-cases/testing-in-staging-environment.md:21]: Add a sentence "Make sure you will use configurationId from the demo environment."

❓

File: distributor-widget/use-cases/connect-to-date-inputs.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (zdrazil) [reply distributor-widget/use-cases/connect-to-date-inputs.md:42]: How about `distributorApi`? So it's more sure the name won't clash with any possible outer `api` variable and reader won't have to know much about JS scopes?

[the remark this answers is not in the document]
Comment (zdrazil) [reply distributor-widget/use-cases/connect-to-date-inputs.md:42]: or ok, let's do `api` it's shorter and not so annoying to type. In real code they will want to use smth shorter like that probably.

File: distributor-widget/use-cases/testing-in-staging-environment.md (hunk 5 above)
Comment (zdrazil) [reply distributor-widget/use-cases/testing-in-staging-environment.md:23]: I've also added a link to FAQ and mentioned that there can be more configuration ids.

File: distributor-widget/getting-started.md [hunk 6]
```diff
@@ -145,7 +145,7 @@ If you want to have a more customized setup, or you want to call some API functi
 
 This callback is later called asynchronously with an argument - Distributor instance. By calling methods on this instance you can control Distributor.
 
-Very common example of this is [using a custom start and end date selectors that are part of your website and then passing user’s selection to Distributor](./use-cases/connect-to-date-inputs.md):
+Very common example of this is [using a custom start and end date selectors that are part of your website and then passing user’s selection to Distributor](./use-cases/use-own-date-inputs.md):
```
Comment (zdrazil) [inline distributor-widget/getting-started.md:148]: Omitted `your` from the link to make it shorter
